docs/documentation/platform/event-subscriptions.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: "Event Subscriptions"
+sidebarTitle: "Events"
+description: "Subscribe to events in Infisical for real-time updates"
+---
+
+<Info>
+ **Note:** Event Subscriptions is a paid feature.
+ - **Infisical Cloud users:** Event Subscriptions is available under the **Enterprise Tier**.
+ - **Self-Hosted Infisical:** Please contact [sales@infisical.com](mailto:sales@infisical.com) to purchase an enterprise license.
+</Info>
+
+Event Subscriptions in Infisical allow you to receive real-time notifications when specific actions occur within your account or organization. These notifications include changes to secrets, users, teams, and other important resources.
```

Comment:
May be just don't say other resources for now or just mark it as coming soon or something like that